### PR TITLE
Removed the repeated team meambers

### PR DESCRIPTION
**Pull request by Ridvmodi**
deployment link : [fossasia.surge.sh](https://fossasia.surge.sh)

### CODE REVIEW

**Comment by Ridvmodi** [comment]
fixed #413 check @mariobehling @hongquan

**Review by sarnava1 — APPROVED** [review]

File: team/index.html [hunk 1]
```diff
@@ -1429,61 +1429,7 @@ <h1>Developers</h1>
...
-									<a href="https://twitter.com/PraveenOjha5"><i class="icon social_twitter"></i></a>
-									<a href="http://praveenojha.me"><i class="icon fa fa-globe"></i></a>
-								</div>
-							</div>
-						</div>
-						<span class="speaker-name">Praveen Ojha</span>
-						<span>Developer</span>
-					</div>
-				</div>
+				<div class="clearfix"></div>
```
Comment (samagragupta) [inline team/index.html]: What is the use of this line?

Comment (Ridvmodi) [reply team/index.html]: I just checked now nothing got affected after removing it, so can I remove it @samagragupta

Comment (Ridvmodi) [reply team/index.html]: Actually this line helps when we add code after that block if we will not add this then there will be some vacant space that's why I didn't remove it @samagragupta

Comment (samagragupta) [reply team/index.html]: Then this <div> could be added when the other code is added.

**Review by pranav1698 — CHANGES_REQUESTED** [review]
Please squash your commits

File: apply/index.html [hunk 2]
```diff
@@ -99,6 +99,7 @@
 											<li><a target="_self" href="https://gci15.fossasia.org">Code-In 2015</a></li>
 											<li><a target="_self" href="https://gci14.fossasia.org">Code-In 2014</a></li>
 										</ul>
+<<<<<<< HEAD:apply/index.html
```
Comment (nikit19) [inline apply/index.html:102]: Please resolve conflicts properly.

**Review by samagragupta — CHANGES_REQUESTED** [review]
Please resolve the conflicts properly.

File: apply/index.html (hunk 2 above)
Comment (championpaddler) [reply apply/index.html:102]: @Ridvmodi  Resolve conflicts .

**Review by championpaddler — CHANGES_REQUESTED** [review]

**Pull request merged**